### PR TITLE
Felinids get cat-tongue (not the organ)

### PR DESCRIPTION
**Pull request by amidoingitright**
It gives felinids a special language like literally every other race in the game

## About The Pull Request

finally some love for felinids, it provides felinids with a special language

## Why It's Good For The Game

this change is good for the game because it rebalances felinids to have a minor language advantage over regular crew, like every other race (excluding humans)

## Changelog
:cl:
add: "Custom" sprite icon for the language
add: Added cat-tongue a language only silicons and felinids can understand
/:cl:

### CODE REVIEW

**Comment by amidoingitright** [comment]
TRAVIS NO HOW COULD YOU DO THIS TO ME?

**Comment by thelaughingbomb** [comment]
You need to add cat-tongue.dm to includes, in alphabetical order

**Comment by amidoingitright** [comment]
why wont this work

**Comment by amidoingitright** [comment]
please travis my sanity is waivering

**Comment by amidoingitright** [comment]
YES YES YES YES, FINALLY I AM FREE

**Comment by amidoingitright** [comment]
any i'm not touching this PR again i'll wait for it to merge before making changes of any kind

File: code/modules/mob/living/carbon/human/species_types/felinid.dm [hunk 1]
```diff
@@ -11,6 +11,7 @@
 	mutanttail = /obj/item/organ/tail/cat
 	changesource_flags = MIRROR_BADMIN | WABBAJACK | MIRROR_PRIDE | MIRROR_MAGIC | RACE_SWAP | ERT_SPAWN | SLIME_EXTRACT
 
+
```
Comment (TheFakeElon) [inline code/modules/mob/living/carbon/human/species_types/felinid.dm:14]: Remove whitespace smh

Comment (amidoingitright) [reply code/modules/mob/living/carbon/human/species_types/felinid.dm:14]: I'll change it on Sunday smh

**Comment by TheFakeElon** [comment]
Possible future conflict: https://github.com/BeeStation/BeeStation-Hornet/pull/2058
Will check when I get home

**Comment by TheFakeElon** [comment]
Yeahh... it’s going to mess up #1712 too

**Comment by MCterra10** [comment]
at that point we can demodularize the majority of the species languages excepting this

**Comment by amidoingitright** [comment]
Aha that's epic

**Comment by amidoingitright** [comment]
i am aware of the bug that felinds dont get the language at round start, i have fixed it but the test merge is yet to adopt it

**Comment by amidoingitright** [comment]
closed because i'm waiting for refactoring to merge

**Comment by MCterra10** [comment]
affirm, removing testmerge.

**Pull request closed without merging**